src/components/Upgrade/content.js

```diff
@@ -146,9 +147,9 @@ export const highlights = [
     description: {
       small: null,
       large: `
-        A glimpse into the future of organizations that
-        you can use today.
-      `,
+          A glimpse into the future of organizations that
+          you can use today.
+        `,
```

Comment:
Should we revert the changes to the indentations?

Reply:
Sure, I'll take care of it in a while as I'm on another branch right now.

Reply:
Done.